rust-mode.el

```diff
@@ -1475,7 +1475,8 @@ This is written mainly to be used as `end-of-defun-function' for Rust."
                (buffer (window-buffer window))
                (start (rust--format-get-pos buffer (pop loc)))
                (pos (rust--format-get-pos buffer (pop loc))))
-          (set-window-start window start)
+          (unless (eq buffer current)
+            (set-window-start window start))
           (set-window-point window pos)))))
 
```

Comment:
Is it possible that `(set-window-point window pos)` is redundant as we already set the position for each buffer in the previous loop ?

Reply:
I am not sure, but I was curious why wrap the call to `set-window-start` but not the one to `set-window-point`.

Reply:
I'm also not sure, but I think `set-window-point` restores a saved location ? `set-window-start` seems to be the reason for  #231 so I only wrapped this function.